gluten-core/src/main/scala/io/glutenproject/execution/WindowExecTransformer.scala

```diff
@@ -68,8 +68,14 @@ case class WindowExecTransformer(
     } else ClusteredDistribution(partitionSpec) :: Nil
   }
 
-  // We no longer require for sorted input for columnar window
-  override def requiredChildOrdering: Seq[Seq[SortOrder]] = Seq.fill(children.size)(Nil)
+  override def requiredChildOrdering: Seq[Seq[SortOrder]] = {
```

Comment:
+1
Gluten replaced SMJ with SHJ, and replace SortAgg with HashAgg, will these bring some similar issue?